File: aspnet-version/backend/PetConnectAlmost/DTOs/ProductDto.cs
namespace PetConnectAlmost.DTOs
{
    public class ProductDto
    {
        public long ProductId { get; set; }
        public string Name { get; set; } = null!;
        public string? Description { get; set; }
        public decimal Price { get; set; }
        public int Quantity { get; set; }
        public bool IsAvailable { get; set; }
        public string? ImageUrl { get; set; }
        public string Category { get; set; } = null!;
    }
}

File: springboot-version/backend/src/main/java/com/petconnect/dto/ProductDto.java
package com.petconnect.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.math.BigDecimal;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ProductDto {
    private Long id;
    private String name;
    private String description;
    private BigDecimal price;
    private int quantity;
    private String imageUrl;
    private boolean isAvailable;
    private Long categoryId;
    private String categoryName;
}
